### PR TITLE
Fake view shopping cart

### LINKED ISSUE #31: Add a "fake" cart view before, with a button to "pay with Credit Card" to see this view
And make it the one to see when the user clicks the "back button"
image

### PR DESCRIPTION
**Pull request by Jesus-Gonzalez-Arroyo**
Vista falsa del carrito con boton para pagar y usar la vista de la tarjeta

fixes #31

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,16 +1,26 @@
...
 import ErrorName from './MessagesError/ErrorName';
 import ErrorNumber from './MessagesError/ErrorNumber';
 import ErrorCode from './MessagesError/ErrorCode';
 import ErrorDate from './MessagesError/ErrorDate';
+import Completed from './MessagesCard/Completed';
+import Incompleted from './MessagesCard/Incompleted';
 
 function App() {
+
+  const Back = useNavigate()
```
Comment (Israel-Laguan) [inline src/App.js:14]: This name is not meaningful, please use another one like this one
```suggestion
  const goTo = useNavigate()
```

File: src/App.js [hunk 2]
```diff
@@ -1,16 +1,26 @@
...
+
+  function BackShoppingCar(){
+    Back('/')
+  }
```
Comment (Israel-Laguan) [inline src/App.js:18]:
```suggestion
  const BackShoppingCar = () => goTo('/');
```

File: src/MessagesCard/Completed.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import '../MessagesCard/styles/Style-MessagesCard.css'
+
+const Completed = () => (
+    <div id="completed" className="completed">
+            <p>Successful payment</p>
+            <a href="/">Your shopping</a>
```
Comment (Israel-Laguan) [inline src/MessagesCard/Completed.js:7]: check this indentation

File: src/shoppingCar/style/car.css [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+    }
+
+    .precioProduct {
+        width: 50%;
+    }
+
+    .pay {
+        width: 100%;
+    }
+}
```
Comment (Israel-Laguan) [inline src/shoppingCar/style/car.css:114]:
```suggestion
}

```

Add EOF line

File: src/shoppingCar/Car.js [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+import '../shoppingCar/style/car.css'
+import {useNavigate} from 'react-router-dom'
+
+function ShoppingCar(){
+
+    const NavButton = useNavigate()
+    
+    function RoutePay(){
+        NavButton('/newcard')
+    }
```
Comment (Israel-Laguan) [inline src/shoppingCar/Car.js:11]:
```suggestion
    const goTo = useNavigate()
    
    const RoutePay = () => goTo('/newcard');
```

File: src/shoppingCar/Car.js [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+                            <p>Nike Force Shoes</p>
+                            <p>Color: black</p>
+                        </div>
+                        <div className="precioProduct">
+                            <p>$305000</p>
+                        </div>
+                        <div className="pay">
+                            <p>I pay</p>
+                            <p>Make your purchase now!</p>
+                            <button onClick={RoutePay}>To pay</button>
```
Comment (Israel-Laguan) [inline src/shoppingCar/Car.js:50]: I think a more "natural" way to do this is using the Link component

https://reactrouter.com/en/main/components/link

```js
<Link to="/newcard">To Pay</Link>
```

File: src/index.js [hunk 7]
```diff
@@ -2,11 +2,18 @@ import React from 'react';
...
 import * as serviceWorker from './serviceWorker';
+import {BrowserRouter, Routes, Route} from 'react-router-dom'
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <BrowserRouter>
+      <Routes>
+        <Route path='/' element={<ShoppingCar/>}/>
+        <Route path='/newcard' element={<App />}/>
```
Comment (Israel-Laguan) [inline src/index.js:14]:
```suggestion
        <Route path='/payment' element={<App />}/>
```

Notice it refers to an action, to pay

File: src/MessagesCard/Completed.js (hunk 3 above)
Comment (Israel-Laguan) [reply src/MessagesCard/Completed.js:7]: I think a more "natural" way to do this is using the Link component

https://reactrouter.com/en/main/components/link

```js
<Link to="/">To Shopping</Link>
```

File: src/App.js [hunk 8]
```diff
@@ -1,16 +1,26 @@
...
+
   return (
     <div className="App">
       <header>
-        <span>&#8249;</span>
+        <span onClick={BackShoppingCar}>&#8249;</span>
```
Comment (Israel-Laguan) [inline src/App.js:23]: I think a more "natural" way to do this is using the Link component

https://reactrouter.com/en/main/components/link

```js
<Link to="/">&#8249;</Link>
```

**Comment by Jesus-Gonzalez-Arroyo** [comment]
Agregando los cambios sugeridos

File: src/App.js [hunk 9]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { useNavigate } from 'react-router-dom';
+import {Link } from 'react-router-dom';
```
Comment (Israel-Laguan) [inline src/App.js:2]:
```suggestion
import { Link } from 'react-router-dom';
```

**Review by Israel-Laguan — APPROVED** [review]

**Pull request merged**